docs/tutorials/ml_inference/semantic_search/bedrock_titan_multi-modal_embedding_model.md

```diff
@@ -0,0 +1,644 @@
+
+- Create connector
+
+```
+POST _plugins/_ml/connectors/_create
+{
+    "name": "Amazon Bedrock Connector: bedrock Titan multi-modal embedding model",
+    "description": "Test connector for Amazon Bedrockbedrock Titan multi-modal embedding model",
+    "version": 1,
+    "protocol": "aws_sigv4",
+    "credential": {
+        "access_key": "{{access_key}}",
+        "secret_key": "{{secret_key}}",
+        "session_token": "{{session_token}}"
+    },
+    "parameters": {
+    "region": "us-east-1",
+    "service_name": "bedrock",
+    "model": "amazon.titan-embed-image-v1",
+    "input_docs_processed_step_size": 2
+    },
+    "actions": [
+               {
+            "action_type": "predict",
+            "method": "POST",
+            "headers": {
+                "content-type": "application/json"
+            },
+            "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model}/invoke",
+            "request_body": "{\"inputText\": \"${parameters.inputText:-null}\", \"inputImage\": \"${parameters.inputImage:-null}\"}"
+        }
+    ]
+}
+```
+Sample response
+```
+{
+  "connector_id": "P8c_JZUB7judm8f4591w"
+}
+```
+
+- Create model
+```
+POST _plugins/_ml/models/_register?deploy=true
+{
+    "name": "amazon.titan-embed-image-v1",
+    "version": "1.0",
+    "function_name": "remote",
+    "description": "test amazon.titan-embed-image-v1",
+    "connector_id": "P8c_JZUB7judm8f4591w",
+    "interface": {
+        "input": "{\n    \"type\": \"object\",\n    \"properties\": {\n        \"parameters\": {\n            \"type\": \"object\",\n            \"properties\": {\n                \"inputText\": {\n                    \"type\": \"string\"\n                },\n                \"inputImage\": {\n                    \"type\": \"string\"\n                }\n            }\n        }\n    }\n}",
+        "output": "{\n    \"inference_results\": [\n        {\n            \"output\": [\n                {\n                    \"name\": {\n                        \"type\": \"string\"\n                    },\n                    \"dataAsMap\": {\n                        \"type\": \"object\",\n                        \"properties\": {\n                            \"embedding\": {\n                                \"type\": \"array\",\n                                \"items\": {\n                                    \"type\": \"number\"\n                                }\n                            },\n                            \"inputTextTokenCount\": {\n                                \"type\": \"number\"\n                            }\n                        }\n                    }\n                }\n            ],\n            \"status_code\": {\n                \"type\": \"integer\"\n            }\n        }\n    ]\n}"
```

Comment:
This output schema is different with our code https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/utils/ModelInterfaceUtils.java#L760

Do you see any issue for the default schema ?

Reply:
this is the output schema without post processing function.